src/db/mormot.db.sql.ibx.pas

```diff
@@ -189,17 +189,29 @@   TSqlDBIbxConnection = class(TSqlDBConnectionThreadSafe)
 
   /// implements a statement via a IBX/FB Pascal API database connection
   TSqlDBIbxStatement = class(TSqlDBStatementWithParamsAndColumns)
+  type
```

Comment:
This code won't compile with Delphi 7 or 2007.

Reply:
I move sub-type, just to inform you MWA fbintf package minimum requirements:
FPC version 3.0.0
Delphi 2010+

Reply:
Ah I didn't know that.
I am not sure the sub-type would compile with Delphi 2010.